sdk/communication/communication-phone-numbers/src/generated/src/phoneNumbersClientContext.ts

```diff
@@ -78,6 +78,6 @@ export class PhoneNumbersClientContext extends coreHttp.ServiceClient {
     this.endpoint = endpoint;
 
     // Assigning values to Constant parameters
-    this.apiVersion = options.apiVersion || "2021-03-07";
+    this.apiVersion = options.apiVersion || "2022-01-11-preview2";
```

Comment:
Generally, we ship beta versions of the package when targeting preview versions of the service. You may want to update the package version to be 1.1.0-beta.1 instead of 1.1.0 unless your team has specific reasons to do otherwise

Reply:
good to know, let check with the team